### ISSUE TITLE
tiles that also show images should provide alt description

### ISSUE THREAD

**Issue by polyester** [collective/collective.cover#182]
Images should have an alt tag, otherwise basic a11y tests go screaming, and it is the right thing to do anyway. Standard Plone news items, for instance, insert the title of the newsarticle as alt tag, if nothing else is available. That is probably a good thing to do for the basic tile as well.
